packages/mui-lab/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/lab",
-  "version": "6.0.0-beta.6",
+  "version": "6.0.0-rc.0",
```

Comment:
Oops this is an experimental package, it should always be in alpha. We likely did a mistake when moving to beta.

Reply:
Changed it to `6.0.0-beta.7` https://github.com/mui/material-ui/pull/43388/files#diff-ebe0a4ae996954db17227ef3a1c0bc820f4a15857165ff08bec0b55ffe73e51aR3

Reply:
It should have always been alpha, but keeping it beta could also make sense. I don't expect we will be updating the package a lot.

Reply:
👍